internal/controller/gameserver_controller.go

```diff
@@ -280,6 +303,12 @@ func (r *GameServerReconciler) reconcilePod(ctx context.Context, gameServer *gam
 							Value: otelResourceAttributes,
 						},
 					},
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "external-ip",
+							MountPath: "/var/run/fellowship/external-ip",
```

Comment:
```suggestion
							MountPath: "/var/run/fellowship",
```
I think otherwise the file will be `/var/run/fellowship/external-ip/external-ip`.

Reply:
ah, thanks!